src/main/steps/sequence.ts

```diff
@@ -195,8 +196,16 @@ export const sequence: Step[] = [
         (data.divorceOrDissolution === DivorceOrDissolution.DISSOLUTION || data.sameSex === Checkbox.Checked)
       ) {
         return RESIDUAL_JURISDICTION;
-      } else if (data.lastHabituallyResident === YesOrNo.NO) {
+      } else if (
+        data.yourLifeBasedInEnglandAndWales === YesOrNo.NO &&
+        data.partnersLifeBasedInEnglandAndWales === YesOrNo.NO &&
+        data.yourDomicileInEnglandWales === YesOrNo.NO &&
+        data.partnersDomicileInEnglandWales === YesOrNo.NO &&
+        data.lastHabituallyResident === YesOrNo.NO
+      ) {
         return JURISDICTION_MAY_NOT_BE_ABLE_TO;
+      } else if (data.lastHabituallyResident === YesOrNo.NO) {
+        return JURISDICTION_CONNECTION_SUMMARY;
```

Comment:
`lastHabituallyResident === YesOrNo.NO`, then going to `JURISDICTION_CONNECTION_SUMMARY` says: `both you and your ${partner} are habitually resident`.

I'm not sure if that matches what they have selected?